[2023-03-29 19:53:52,508] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T19:53:51.878228+00:00 [queued]>
[2023-03-29 19:53:52,517] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T19:53:51.878228+00:00 [queued]>
[2023-03-29 19:53:52,517] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 19:53:52,517] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 19:53:52,517] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 19:53:52,531] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): start_task> on 2023-03-29 19:53:51.878228+00:00
[2023-03-29 19:53:52,535] {standard_task_runner.py:52} INFO - Started process 209 to run task
[2023-03-29 19:53:52,537] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'start_task', 'manual__2023-03-29T19:53:51.878228+00:00', '--job-id', '684', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmpo1t4s25f', '--error-file', '/tmp/tmpmg_6ii2v']
[2023-03-29 19:53:52,537] {standard_task_runner.py:80} INFO - Job 684: Subtask start_task
[2023-03-29 19:53:52,588] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.start_task manual__2023-03-29T19:53:51.878228+00:00 [running]> on host fe0c70026834
[2023-03-29 19:53:52,656] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=start_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T19:53:51.878228+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T19:53:51.878228+00:00
[2023-03-29 19:53:52,656] {logging_mixin.py:115} INFO - DAG has been started...
[2023-03-29 19:53:52,656] {python.py:173} INFO - Done. Returned value was: DAG has been started...
[2023-03-29 19:53:52,681] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=start_task, execution_date=20230329T195351, start_date=20230329T195352, end_date=20230329T195352
[2023-03-29 19:53:52,709] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 19:53:52,752] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
